spec.html

```diff
@@ -7679,7 +7691,7 @@ <h1>AsyncGeneratorFunctionCreate (_kind_, _ParameterList_, _Body_, _Scope_, _Str
       </emu-alg>
     </emu-clause>
 
-    <!-- es6num="9.2.7" -->
+    <!-- es6num="9.2.8" -->
```

Comment:
Please revert all of these es6num changes; these are legacy links from ES6 and should never be renumbered.

Reply:
No, things that weren’t in es6 shouldnt have es6nums